tests/kernel/workq/work_queue/src/main.c

```diff
@@ -158,8 +166,10 @@ static void resubmit_work_handler(struct k_work *work)
 		k_work_submit(work);
 	}
 }
+
 /**
- * @ingroup kernel_workqueue_tests
```

Comment:
this is correct, adding to a group can be done in multiple ways. Just add the @brief below it. same for all functions below.

Please verify your changes by examining the doxygen output